### DISCUSSION TITLE
How do i run this on a mac

### DISCUSSION THREAD

**Discussion by dusanyu** [cinderblocks/radegast#59] [General]
I installed  dotnet and it says i am missing Redigast.runtimeconfig.jason i am by no means a microsoft  person so where is this file? It would make sense to run this on a M2 mini that uses less power than a full windows PC

**Comment by cinderblocks** [comment]
Mono's Carbon driver has never been ported to 64-bit. WinForms is very unlikely to work if you are able to run it. The most effective way to run Radegast on a Mac at this point is under Wine or Crossover emulation.
